### ISSUE TITLE
Assigning colours to pre-defined groups

### ISSUE THREAD

**Issue by JMarzec** [romanhaa/cerebroApp#29]
Hi there, 

First of all, huge thanks for developing Cerebro! We find it super useful too in our lab 🙂

Is it possible to assign colours to pre-defined groups in Cerebro? Like you do for the Cell Cycle groups (**G1** = `#45aaf2`, **S** - `#f1c40f`, **G2M** = `#e74c3c` in `cerebroApp/shiny/v1.3/color_setup.R`)

Or where can we add these is the shiny scripts? I was looking at `cerebroApp/shiny/v1.3/color_setup.R ` and `cerebroApp/shiny/v1.3/color_management/server.R` but I couldn't figure it out.

Many thanks for help and this fab work!

**Comment by romanhaa** [comment]
Hi Jacek,

Thank you for the kind words! It sounds like your question is very similar to [this](https://github.com/romanhaa/cerebroApp/issues/22#issuecomment-758474471).

Let me know if that helps.

Please note that, sadly, I decided to stop actively developing Cerebro. It was not an easy decision but as the only developer, in the quickly developing field of scRNA-seq analysis, and not being in academia anymore, I simply don't have the resources to continue improving or even maintaining it. Feedback from users who enjoy using Cerebro for their research, like you, have been the primary motivation to make and improve Cerebro. That is exactly what it was made for. I would be happy to continue as one of multiple contributors, with Cerebro being a community project, but I can't do it alone.

Cheers,
Roman

**Comment by JMarzec** [comment]
Hi Roman,

Thank you for response. First of all, I completely understand your decision and appreciate all your hard work on Cerebro so far. This app is great and has grown so much that community suport sounds like the way to go.

Thank you also for pointing to the relevant question! The solution [there](https://github.com/romanhaa/cerebroApp/issues/22) worked for me partially. I've found that modyfing `assignColorsToGroups()` in the `utility_functions.R` file changes colors only in the UMAP/scatter plots. However, it was great start for solving my problem as it point me to the `color_setup.R` script that I've modified accordingly and this has changed colours in "Color management" tab as well as in all the other plots and tables in other tabs! I'll provide more details within [issue 22](https://github.com/romanhaa/cerebroApp/issues/22).

Once again huge thanks for developing Cerebro and help with solving my issues.
Best
Jacek
